src/core/mac/sub_mac.hpp

```diff
@@ -625,7 +615,6 @@ class SubMac : public InstanceLocator, private NonCopyable
     static constexpr uint32_t kCslReceiveTimeAhead = OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD;
 
     enum CslState : uint8_t{
```

Comment:
Can now remove the `enum CslState` definition?

Reply:
Somehow I missed to remove this one.